### DISCUSSION TITLE
Printable cheat sheet

### DISCUSSION THREAD

**Discussion by Dinock** [GDQuest/learn-gdscript#138] [Ideas]
In addition to the upcoming glossary a cheat sheet/crib sheet/reference card of one or two pages could be helpful. 
Beginners could print this sheet out and hang it up their wall to have a quick look while coding or even when the computer is turned off. This could help to memorize how to define a function for example without the need to look it up in the glossary all the time.

**Comment by NathanLovato** [comment]
We plan to do that as part of our paid course Learn to Code from Zero with Godot, so we'll likely not provide this in this repository.
